statistics/queries/continent_histogram.sql

```diff
@@ -82,10 +82,9 @@ dl_histogram AS (
     CASE WHEN bucket_left = 0.31622776601683794 THEN 0
     ELSE bucket_left END AS bucket_min,
     bucket_right AS bucket_max,
-    COUNTIF(download_MED < bucket_right AND download_MED >= bucket_left) AS dl_samples_bucket,
-    COUNT(*) AS dl_samples_day,
-    COUNTIF(download_MED < bucket_right AND download_MED >= bucket_left) / COUNT(*) AS dl_frac
-  FROM dl_stats_perip_perday CROSS JOIN buckets
+    COUNTIF(stats.download_MED < bucket_right AND stats.download_MED >= bucket_left) AS dl_samples_bucket,
+    ROUND(COUNTIF(stats.download_MED < bucket_right AND stats.download_MED >= bucket_left) / COUNT(*), 3) AS dl_frac_bucket
+  FROM dl_random_perip_perday CROSS JOIN buckets
```

Comment:
The histogram should use the random rows for each IP/day/geo (mbps, not mbps_MED)

Reply:
Done in [d516ad7](https://github.com/m-lab/stats-pipeline/pull/42/commits/d516ad7bdad668b0b71f348e532838ab21b1c438)